packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsInfo/PayableDetailsInfo.tsx

```diff
@@ -64,6 +75,26 @@ const StyledLabelTableCell = styled(TableCell)(({ theme }) => ({
   width: '35%',
 }));
 
+const RequiredFieldLabel = ({
```

Comment:
I would prefer to keep all child components in the end of the file. Its much easier when you read the code to start with the main parts.